# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ktcpvs/make_install.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2004-12-17 14:37:39.000000000 -0300
+++ ./Makefile	2005-04-07 23:06:46.043750480 -0400
@@ -22,7 +22,8 @@
 # relocations required by build roots.  This is not defined in the
 # makefile but the arguement can be passed to make if needed.
 
-MODLIB		= $(INSTALL_MOD_PATH)/lib/modules/$(KERNELRELEASE)/kernel/net/ktcpvs/
+MODDIR		= kernel/net/ktcpvs/
+MODLIB		= $(INSTALL_MOD_PATH)/lib/modules/$(KERNELRELEASE)/$(MODDIR)
 
 # If System.map exists, run depmod.  This deliberately does not have a
 # dependency on System.map since that would run the dependency tree on
@@ -36,6 +37,7 @@
 endif
 
 CC		= gcc
+DEPMOD		= /sbin/depmod
 CFLAGS		= -D__KERNEL__ -DMODULE  -DEXPORT_SYMTAB -DMODVERSIONS	\
 		$(SMPFLAGS) $(DEBUGFLAGS) -O2 -Wall			\
 		-Wstrict-prototypes -I$(KERNELSOURCE)/include		\
@@ -97,7 +99,7 @@
 modules_install: $(TARGETS)
 		if [ ! -d "$(MODLIB)" ]; then mkdir -p "$(MODLIB)"; fi
 		install -m 600 -c $(TARGETS) "$(MODLIB)"
-		if [ `/usr/bin/id -u` = '0' ]; then depmod -ae -F $(KERNELSYMS) $(depmod_opts) $(KERNELRELEASE); fi
+		if [ `/usr/bin/id -u` = '0' ]; then $(DEPMOD) -ae -F $(KERNELSYMS) $(depmod_opts) $(KERNELRELEASE); fi
 
 modules_uninstall:
 		(cd "$(MODLIB)"; rm -f $(TARGETS))
